src/lib/mutate-request.ts

```diff
@@ -42,45 +42,45 @@ export class PeprMutateRequest<T extends KubernetesObject> {
     }
   }
 
-  Merge = (obj: DeepPartial<T>) => {
+  Merge = (obj: DeepPartial<T>): void => {
     this.Raw = mergeDeepRight(this.Raw, obj) as unknown as T;
   };
 
-  SetLabel = (key: string, value: string) => {
+  SetLabel = (key: string, value: string): this => {
     const ref = this.Raw;
     ref.metadata = ref.metadata ?? {};
     ref.metadata.labels = ref.metadata.labels ?? {};
     ref.metadata.labels[key] = value;
     return this;
   };
 
-  SetAnnotation = (key: string, value: string) => {
+  SetAnnotation = (key: string, value: string): this => {
```

Comment:
This is equivalent to saying `PeprMutateRequest<T extends KubernetesObject>`. That said, a quick google suggests:

> In TypeScript, you can use the `this` type as a return type to indicate that a method returns the instance of the class it belongs to. This is particularly useful for creating fluent interfaces or method chaining.

Just wanted to call it out here, I can accept either way.

Reply:
good to know, i noticed the linter was complaining about it but I admit it looks strange